src/components/forms/AddEmployeeForm.js

```diff
@@ -69,23 +118,19 @@ function AddEmployeeForm({ visible, onCreate, onCancel }) {
         >
           <Select size="large" placeholder="Select Employee Role">
             {/* Could be dynamic by mapping through list of roles */}
-            <Select.Option value="Administrator">Administrator</Select.Option>
-            <Select.Option value="Program Manager">
-              Program Manager
-            </Select.Option>
-            <Select.Option value="Service Provider">
-              Service Provider
-            </Select.Option>
+            <Select.Option value={1}>Administrator</Select.Option>
```

Comment:
Good catch!! We know the BE needs to be passed numerical values instead of strings for the provider roles.